Development/src/App.js

```diff
@@ -12,83 +13,40 @@ import { EventsList, EventsShow } from './pages/logs';
 import { SubscriptionsList, SubscriptionsShow } from './pages/subscriptions';
 import { QueryAPIList, QueryAPIShow } from './pages/queryapis';
 import dataProvider from './dataProvider';
-
-import AnnouncementIcon from '@material-ui/icons/Announcement';
-import CallMadeIcon from '@material-ui/icons/CallMade';
-import CallReceivedIcon from '@material-ui/icons/CallReceived';
-import SettingsCellIcon from '@material-ui/icons/SettingsCell';
-import SettingsInputAntennaIcon from '@material-ui/icons/SettingsInputAntenna';
-import CompareArrowsIcon from '@material-ui/icons/CompareArrows';
-import DvrIcon from '@material-ui/icons/Dvr';
-import SwapVertIcon from '@material-ui/icons/SwapVert';
-import SettingsIcon from '@material-ui/icons/Settings';
-import SettingsAppIcon from '@material-ui/icons/SettingsApplications';
-import { useTheme } from '@material-ui/styles';
-
 import MyLayout from './MyLayout';
 
 const App = () => (
     <Admin appLayout={MyLayout} dataProvider={dataProvider} theme={useTheme()}>
-        <Resource name="Settings" list={Dashboard} icon={SettingsIcon} />
+        <Resource name="Settings" list={Dashboard} />
```

Comment:
Icons are defined the sidebar list items instead?

Reply:
Correct
0b685b1

Reply:
Correct
0b685b1